.github/ISSUE_TEMPLATE/dataset_template.md

```diff
@@ -4,23 +4,29 @@ about: Suggest a new dataset for Tidy Tuesday
 labels: dataset
 ---
 
-Please fill out as much of this information as you can! 
+Please consider [submitting this dataset as a pull request](pr_instructions.md)!
 
-- **Dataset name:** 
-- **Dataset download URL:** 
-- **Article that demonstrates the dataset:**
-- **Image 1 URL:**
-- **Image 1 ALT text:**
-- **Image 2 URL:**
-- **Image 2 ALT text:**
+Please fill out as much of this information as you can!
 
-**Cleaning script:**
+- **title:** A short name for the dataset (to be used in the social media post, "This week we're exploring DATASET NAME"). Examples: "Canadian NHL Player Birth Dates", "R Consortium ISC Grants", or "Leap Day".
+- **article:** An example of the dataset being used, such as a blog post or a README about the dataset.
+  - **title:** The title of the article.
+  - **url:** The link to the article.
+- **data_source:** A source where the dataset can be downloaded.
+  - **title:** The title of the source.
+  - **url:** The link to the source.
+- **images:** One or more images related to the dataset. For each image, provide:
+  - **file:** A url to download the image, or an attached file.
+  - **alt:** Text that can serve as a replacement for the image for those who cannot see the image (whether through visual impairment or because the image does not load).
```

Comment:
```suggestion
- [ ] This dataset has not already been used in TidyTuesday.
- [ ] The dataset will (probably) be less than 50MB when saved as a tidy CSV.
- [ ] I can imagine a data visualization related to this dataset.

- [ ] **title:** A short name for the dataset (to be used in the social media post, "This week we're exploring DATASET NAME"). Examples: "Canadian NHL Player Birth Dates", "R Consortium ISC Grants", or "Leap Day".
- [ ] **article:** An example of the dataset being used, such as a blog post or a README about the dataset.
  - [ ] **title:** The title of the article.
  - [ ] **url:** The link to the article.
- [ ] **data_source:** A source where the dataset can be downloaded.
  - [ ] **title:** The title of the source.
  - [ ] **url:** The link to the source.
- [ ] **images:** One or more images related to the dataset. For each image, provide:
  - [ ] **file:** A url to download the image, or an attached file.
  - [ ] **alt:** Text that can serve as a replacement for the image for those who cannot see the image (whether through visual impairment or because the image does not load).
```